fetch_from_legacy.py

```diff
@@ -60,6 +60,12 @@ def handle_endtag(self, tag):
 context.check_hostname = False
 context.verify_mode = ssl.CERT_NONE
 
+# Extract out username/password from index_url, if present.
```

Comment:
This would break usernames/passwords coming from netrc.

The correct solution is probably 
``` python
...
username = cleaned_url.username or username
password = cleaned_url.password or password
...
```

Reply:
@adisbladis thanks for pointing this out. Updated!